web/src/components/common/DataTable.jsx

```diff
@@ -351,42 +351,57 @@ export default class BookmarksTable extends Component {
         }
       })
     }
-    if(this.props.columns.indexOf("archive") !== -1){
+    if(this.props.columns.indexOf("dearchive") !== -1){
       columns.push({
-        Header: 'Archive',
+        Header: 'Unarchive',
```

Comment:
I think we should call it either Dearchive or Unarchive, but not sometimes the first and other times the other